generators/server/templates/src/test/kotlin/package/web/rest/AccountResourceIT.kt.ejs

```diff
@@ -172,30 +171,34 @@ class AccountResourceIT <% if (databaseType === 'cassandra') { %>: AbstractCassa
 
     @BeforeEach
     fun setup() {
-        val accountUserMockResource = new AccountResource()
+        val accountUserMockResource = AccountResource()
         this.mockMvc = MockMvcBuilders.standaloneSetup(accountUserMockResource)
-                .setControllerAdvice(exceptionTranslator)
-                .build()
+            .setControllerAdvice(exceptionTranslator)
+            .build()
     }
 
     @Test
     @Throws(Exception::class)
-    @WithMockUser(username = "test", roles = "ADMIN")
+    @WithMockUser(username = "test", roles = ["ADMIN"])
     fun testGetExistingAccount() {
-        mockMvc.perform(get("/api/account")
-                .accept(MediaType.APPLICATION_JSON))
-                .andExpect(status().isOk())
-                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
-                .andExpect(jsonPath("$.login").value("test"))
-                .andExpect(jsonPath("$.authorities").value(ADMIN))
+        mockMvc.perform(
```

Comment:
Can you check the alignment here?

Reply:
This the auto-formatting done by IntelliJ IDEA.